test-exports-cjs/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "test-exports-cjs",
+  "version": "0.0.0",
+  "private": true,
+  "description": "CJS Tests for the things exported by the langchain package",
+  "main": "./index.mjs",
+  "scripts": {
+    "test": "npm run test:esm && npm run test:cjs && npm run test:cjs:import && npm run test:ts",
+    "test:esm": "node ./index.mjs",
+    "test:cjs": "node ./require.js",
+    "test:cjs:import": "node ./import.js",
+    "test:ts": "tsc && node dist/index.js",
+    "format": "prettier --write \"**/*.ts\"",
+    "format:check": "prettier --list-different \"**/*.ts\""
```

Comment:
consider adding `test:format` that runs `prettier --check` like this

also same feedback w/ using `npm-run-all` (or a similar alternative to the `&&` approach)